sdk/core/azure-core-tracing-opentelemetry/test/ut/service_support_test.cpp

```diff
@@ -670,7 +669,7 @@ TEST_F(OpenTelemetryServiceTests, ServiceApiImplementation)
     "http.method": "GET",
     "http.url": "https://www.microsoft.com",
     "requestId": ".*",
-    "http.user_agent": "MyApplication azsdk-cpp-Azure.Core.OpenTelemetry.Test.Service/1.0.0.beta-2.*",
+    "http.user_agent": "MyApplication azsdk-cpp-Azure-Core-OpenTelemetry-Test-Service/1.0.0.beta-2.*",
```

Comment:
As far as I can tell, the official packages follow the Azure guidelines. I'm not as picky about test collateral, because it never appears on the wire. The only thing that matters here is ensuring that the input and output match.